rotkehlchen/db/updates.py

```diff
@@ -154,6 +155,53 @@ def update_rpc_nodes(self, to_version: int) -> None:
             new_default_nodes=new_default_nodes,
         )
 
+    def update_contracts(self, to_version: int) -> None:
+        """Updates evm contracts and ABIs in globaldb"""
+        updated_contracts: dict
+        success, updated_contracts = self._get_file_content(  # type: ignore[assignment]  # updated_contracts is a dictionary for sure  # noqa: E501
+            file_url=CONTRACTS_URL.format(branch=self.branch, version=to_version),
+            update_type=UpdateType.CONTRACTS,
+        )
```

Comment:
this is ugly because is breaking the types. `_get_file_content` expects to return a list of dicts. You should overload it if is not the case and adjust properly the type